default/stringtables/en.txt

```diff
@@ -6283,6 +6295,21 @@ EFFECT_BIOWEAPON
 EFFECT_BIOWEAPON_LABEL
 Bioweapon Attack
 
+EFFECT_PLANET_TARGETED
+At %planet%: our planet is being targeted by a planet destroyer cannon from the ship %ship%!
+
+EFFECT_PLANET_DESTRUCTION
+We have destroyed the %empire% planet %planet%!
+
+EFFECT_PLANET_DESTRUCTION_LABEL
+Destruction of hostile planet
+
+EFFECT_PLANET_DESTRUCTION_OWN
+Our planet %planet% was destroyed by %empire%!
+
+EFFECT_PLANET_DESTRUCTION_OWN_LABEL
+Destruction of own planet
```

Comment:
When is this used? If it's a sitrep it needs to say where.

Reply:
It's the label/category, as opposed to "Destruction of hostile planet".

Reply:
Of course it is, sorry, it even says _LABEL. I need more coffee.

Reply:
That depends. Maybe the problem is that you actually had too much... 😉